cv-data-service-library/src/main/java/com/trihydro/library/helpers/TimGenerationHelper.java

```diff
@@ -523,23 +553,27 @@ public List<ResubmitTimException> expireTimAndResubmitToOde(List<Long> activeTim
                 List<Milepost> allMps = getAllMps(wydotTim);
                 if (allMps.size() < 2) {
                     String exMsg = String.format(
-                            "Unable to resubmit TIM, less than 2 mileposts found for Active_Tim %d",
-                            tum.getActiveTimId());
+                        "Unable to resubmit TIM, less than 2 mileposts found for Active_Tim %d",
+                        tum.getActiveTimId());
                     utility.logWithDate(exMsg);
                     exceptions.add(new ResubmitTimException(activeTimId, exMsg));
                     continue;
                 }
-                Milepost firstPoint = allMps.get(0);
-                Milepost secondPoint = allMps.get(1);
+
+                Milepost anchorMp = attemptToCalculateAnchorPoint(tum, exceptions, allMps);
+                if (anchorMp == null) {
+                    return;
+                }
```

Comment:
Clarification (non-blocking): This part of the code now invokes the attemptToCalculateAnchorPoint method, which can handle the IdenticalPointsException.